### PR TITLE
VX-1374 update to use scss

### PR DESCRIPTION
**Pull request by littlespice33**

### CODE REVIEW

**Review by michaelmcauley — APPROVED** [review]

File: angular.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
               "src/assets"
```
Comment (cpeisert) [inline angular.json:102]: Line 99: `inlineStyleLanguage` should be `scss`

File: angular.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
             ],
             "styles": [
-              "src/styles.sass"
```
Comment (cpeisert) [inline angular.json:37]: Line 13: `style` should be `scss`
Line 31: `inlineStyleLanguage` should be `scss`

**Review by cpeisert — COMMENTED** [review]
Just a few references to `sass` should be flipped to `scss`. Surprisingly, Angular didn't complain with running the app and updating the scss file.

File: angular.json (hunk 1 above)
Comment (cpeisert) [reply angular.json:102]: Note: GitHub wouldn't let me add these comments at the unchanged lines. See: https://github.com/github/roadmap/issues/347

Comment (littlespice33) [reply angular.json:102]: nice callout. Thanks!

**Review by cpeisert — APPROVED** [review]
LGTM

**Pull request merged**